### PR TITLE
Update interventions to have distinct param and state.

### PR DESCRIPTION
**Pull request by Tom-Szendrey**
# Description

The interfaces in ciemss utilize different inputs for state and parameter interventions.
https://github.com/ciemss/pyciemss/blob/main/pyciemss/interfaces.py

This PR addresses this and corrects the converter in pyciemss-service.

image

### CODE REVIEW

File: service/models/operations/optimize.py [hunk 1]
```diff
@@ -166,6 +167,7 @@ def gen_pyciemss_args(self, job_id):
             "bounds_interventions": self.bounds_interventions,
             "static_parameter_interventions": optimize_interventions,
             "fixed_static_parameter_interventions": fixed_static_parameter_interventions,
+            # "fixed_static_state_interventions": fixed_static_state_interventions, Does not exist?
```
Comment (dgauldie) [inline service/models/operations/optimize.py:170]: what's this commented-out line for?

**Review by dgauldie — APPROVED** [review]
LGTM

File: service/models/converters.py [hunk 2]
```diff
@@ -44,15 +44,22 @@ def fetch_and_convert_dynamic_interventions(policy_intervention_id, job_id):
...
+            if inter.type == "parameter":
+                time = torch.tensor(float(static_inter.timestep))
+                parameter_name = inter.applied_to
+                value = torch.tensor(float(static_inter.value))
+                static_param_interventions[time][parameter_name] = value
+            if inter.type == "state":
+                time = torch.tensor(float(static_inter.timestep))
+                parameter_name = inter.applied_to
+                value = torch.tensor(float(static_inter.value))
+                static_state_interventions[time][parameter_name] = value
```
Comment (shawnyama) [inline service/models/converters.py:61]: what's the point of this, dont they do the same thing?

Comment (Tom-Szendrey) [reply service/models/converters.py:61]: Need to filter states into one variable and parameters into another otherwise ciemss gets confused

**Review by shawnyama — APPROVED** [review]

**Pull request merged**